api/query/cache/service/main.go

```diff
@@ -313,6 +313,12 @@ func main() {
 	// after backfilling was started.
 	go poll.KeepRunsUpdated(store, logger, *updateInterval, *updateMaxRuns, idx)
 
+	var netClient = &http.Client{
+		Timeout: time.Second * 5,
+	}
+	// Polls Metadata update every 10 minutes.
+	go shared.KeepMetadataUpdated(netClient, logger, time.Second*10)
```

Comment:
```suggestion
	go shared.KeepMetadataUpdated(netClient, logger, time.Minute*10)
```

Also this probably shouldn't live in `shared` as it's only used by searchcache.